ostrica/Plugins/BlackLists/__init__.py

```diff
@@ -21,13 +21,25 @@
 #				along with OSTrICa. If not, see <http://www.gnu.org/licenses/>.
 #-------------------------------------------------------------------------------
 import sys
-import httplib
-import string
-import socket
+if sys.version_info < (3, 0):
+  import httplib
+  import StringIO
+else:
+  import http.client as httplib
+  import io as StringIO
 import gzip
-import re
-import StringIO
-from bs4 import BeautifulSoup
+
+def get_zip_obj(data):
+  if sys.version_info < (3, 0):
+    return StringIO.StringIO(data)
+  else:
+    return StringIO.BytesIO(data)
+  return data
+
+def str_if_bytes(data):
+  if type(data) == bytes:
```

Comment:
You can probably do something like:
return data.decode('utf-8') if type(data) == bytes else data

It looks better. Please have a look at the other codes which looks similar.